app/DynamicsAdapter/DynamicsAdapter.Web/appsettings.json

```diff
@@ -7,6 +7,32 @@
     }
   },
   "AllowedHosts": "*",
+  "AppSettings": {
```

Comment:
The configuration in the docker-compose needs to be updated for the environment variables to `APPSETTINGS__SCHEDULERCONFIG__CRON`

Reply:
Not sure I understand this.. are we using environment variables and not app settings file

Reply:
AppSettings can be overriden with env variables so that in Openshift they can configure all of that